include/dt-bindings/interrupt-controller/arm-gic.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2018 Lexmark International, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef __DT_BINDING_ARM_GIC_H
+#define __DT_BINDING_ARM_GIC_H
+
+/* CPU Interrupt numbers */
+#define	GIC_INT_VIRT_MAINT		25
```

Comment:
These values seem like the would be platform/SoC specific?

Reply:
They are called out in the general GIC400 datasheet, http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.ddi0471b/index.html section 2.3.2...